container/CrabV2/Auction/AuctionBody.tsx

```diff
@@ -249,6 +254,16 @@ const BidForm: React.FC = () => {
 
   const canPlaceBid = auctionStatus === AuctionStatus.LIVE || auctionStatus === AuctionStatus.UPCOMING
 
+  const warning = useMemo(() => {
+    const _estPrice = convertBigNumber(estClearingPrice, 18)
+    if ((auction.isSelling && Number(price) < _estPrice) || (!auction.isSelling && Number(price) > _estPrice)) {
+      return `You are quoting a worse price than the est. clearing price: ${_estPrice.toFixed(4)}`
+    }
+    if (auctionStatus === AuctionStatus.UPCOMING) {
+      return 'Auction not started yet. If the price not matched, bid will be cancelled'
```

Comment:
wdym price not matched? Also we might want to say order so we don't have to update bid / offer based on direction